rt_dashboard/web.py

```diff
@@ -257,7 +257,9 @@ def list_jobs(queue_name, page):
         )
 
         offset = (current_page - 1) * per_page
-        jobs = [serialize_job(job) for job in queue.get_tasks(offset, per_page)]
+        jobs = sorted(
+            (serialize_job(job) for job in queue.get_tasks(offset, per_page)),
+            key=itemgetter('enqueued_at'), reverse=True)
```

Comment:
You realize that this only changes the sorting on the current page, right? But it is wrong at the moment?

Reply:
Right, we need to actually get_tasks() in the right order